src/main/java/com/code_intelligence/jazzer/mutation/mutator/proto/BuilderAdapters.java

```diff
@@ -43,11 +43,19 @@ public int size() {
       }
 
       @Override
-      public boolean add(U u) {
-        builder.addRepeatedField(field, u);
+      public boolean add(U element) {
+        builder.addRepeatedField(field, element);
         return true;
       }
 
+      @Override
+      public void add(int index, U element) {
+        // TODO: Simply appending and ignoring the index is not perfect
```

Comment:
Could you file a follow-up ticket for this?

Reply:
done